src/index.html

```diff
@@ -5,8 +5,317 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          class="header__img"
+          src="images/logo.png"
+          alt="Logo">
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+            Laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="cards-block">
+      <div class="cards-block__all-cards">
+        <div class="card" data-qa="card">
+          <img
+            class="card__photo"
+            src="./images/imac.jpeg"
+            alt="Retina 5K Monoblock">
+          <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__reviews">
+            <div class="stars card__stars-block">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__container-review">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <h2 class="card__price-word">Price:</h2>
+            <p class="card__digital-price">$2,199</p>
+          </div>
+          <a
+            class="card__button-link"
+            href="#"
+            data-qa="card-hover"
+          >
+            BUY
```

Comment:
```suggestion
            Buy
```
add `text-transform: uppercase;` instead and remove default underline